packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -277,17 +278,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   }
 
   protected renderSuggestions(): JSX.Element | null {
-    const TypedSuggestions = this.SuggestionOfProperType;
-
-    // TODO:
-    // Move this styled component in a separate file and make it available to the public API.
-    // This should be done after rewriting pickers to use a composition pattern instead of inheritance.
-    const StyledTypedSuggestions = styled<ISuggestionsProps<T>, ISuggestionsStyleProps, ISuggestionsStyles>(
-      TypedSuggestions,
-      suggestionsStyles,
-      undefined,
-      { scope: 'Suggestions' }
-    );
+    const StyledTypedSuggestions: React.ComponentClass<ISuggestionsProps<T>> = StyledSuggestions as any;
```

Comment:
don't love the use of `any` without an explanation here.

Does typing StyledSuggestions as `React.ComponentClass<ISuggestionsProps<any>>` work here instead of going through `any` to tighten the type, or does the variance on the types not work out?

Reply:
I moved the `any` part to `styled` method invoke place.